ui/plugins/eu.esdihumboldt.hale.ui/src/eu/esdihumboldt/hale/ui/service/project/internal/ProjectServiceImpl.java

```diff
@@ -884,8 +885,8 @@ public void run() {
 
 	@Override
 	public void update() {
-		// no change check as this is done by clean before a new project is
-		// loaded
+		// check if there are changes, as you are about to load the same project
```

Comment:
Related to the original comment, how is the behavior in this case?

1. open a project
2. modify it
3. new alignment project

Reply:
You will get this question
![image](https://github.com/halestudio/hale/assets/67077116/a7ff41eb-7db0-4fed-aa3a-7189243c78c1)
and here depends by the user and his answer if the changes will be saved or not.

Reply:
Also a question related to the issue (#956) - the description says the possibility to save the project was offered, but the changes were not saved. Is that also the behavior you observed?
The main task of the `changeCheck` method seems to be to show that dialog, that was also shown before according to the issue description. So does your change solve this by showing it at an earlier time and the later point where it was already checked does not result in the dialog being shown?

Reply:
> Also a question related to the issue (#956) - the description says the possibility to save the project was offered, but the changes were not saved. Is that also the behavior you observed?

I do observe the same behaviour. 

> The main task of the changeCheck method seems to be to show that dialog, that was also shown before according to the issue description. So does your change solve this by showing it at an earlier time and the later point where it was already checked does not result in the dialog being shown?

It is exactly how you say. The solution solves the problem showing the dialog earlier. The dialog is not showed anymore later because the changes has ben saved so when checking if there is any change the clean method exit immediately without showing again the dialog.